tests/unit/cmd/scan/test_path.py

```diff
@@ -216,6 +216,19 @@ def test_scan_ignored_file(self, scan_mock, cli_fs_runner):
         assert "An ignored file or directory cannot be scanned." in result.stdout
         scan_mock.assert_not_called()
 
+    @pytest.mark.parametrize("verify", [True, False])
```

Comment:
Do we care about the case where verify is False? I think it could be more useful to test all possible positions of `--allow-self-signed`.